packages/builder/builder-doc/docs/en/guide/advanced/rspack-start.mdx

```diff
@@ -15,14 +15,14 @@ Of course, you can also access it manually by following the steps below:
 1. Install rspack-provider（The version needs to be the same as the modern.js version).
 
 ```
-pnpm add @modern-js/builder-rspack-provider@2.4.0 -D
+pnpm add @modern-js/builder-rspack-provider@2.8.0 -D
```

Comment:
How about remove the fixed version

Reply:
This is not necessary, but I think it is better to update to a newer version

Reply:
Why not just remove the version?

Reply:
ok